### PR TITLE
add docker instructions

### PR DESCRIPTION
**Pull request by mpmdean**
This is an update to the docs addressing #99

### CODE REVIEW

File: docs/source/user/usedocker.rst [hunk 1]
```diff
@@ -59,3 +59,14 @@ Connect to docker python session with Jupyter
   
   Jupyterlab provides access to a unix terminal, which you can create via File > New > Terminal. We suggest changing the terminal shell type by issuing the command ``bash`` in the terminal.
   
+
+Sharing your code
+-----------------
+
+Using Docker is a nice way to straightforwardly share your code with others. The standard way to specify which docker image is needed to run your code is to include a file named dockerfile with the following contents ::
```
Comment (mrakitin) [inline docs/source/user/usedocker.rst:66]:
```suggestion
Using Docker is a nice way to straightforwardly share your code with others. The standard way to specify which docker image is needed to run your code is to include a file named ``Dockerfile'' with the following contents ::
```

**Review by mrakitin — COMMENTED** [review]
I spotted a minor issue with the `Dockerfile` name. Suggested a fix below.

File: docs/source/user/usedocker.rst (hunk 1 above)
Comment (mpmdean) [reply docs/source/user/usedocker.rst]: Good spot. Thanks. I've changed it.

Comment (mpmdean) [reply docs/source/user/usedocker.rst]: This should be resolved. I think the marks around Dockerfile should be two pairs of backticks?

Comment (mrakitin) [reply docs/source/user/usedocker.rst]: Oh, yes, good catch. Mixed it with LaTeX.

**Pull request merged**